crates/rslint_core/src/groups/errors/constructor_super.rs

```diff
@@ -0,0 +1,92 @@
+use crate::rule_prelude::*;
+use ast::{ClassDecl, ClassElement, Expr, Stmt};
+
+declare_lint! {
+    /**
+    Verify calls of `super()` in constructors (constructor-super)
+
+    The `"extends": "rslint:recommended"` property in a configuration file enables this rule.
+
+    Constructors of derived classes must call `super()`. Constructors of non derived classes must not call `super()`. If this is not observed, the JavaScript engine will raise a runtime error.
```

Comment:
```suggestion
    Constructors of derived classes must call `super()`. Constructors of non derived classes must not call `super()`. 
    If this is not observed, the JavaScript engine will raise a runtime error.
```
Just making the line length a bit smoler